src/jade/templates/todo-list.jade

```diff
@@ -0,0 +1,10 @@
+
+//-
+//- Copyright (C) 2015 by Looker Data Services, Inc.
+//- All rights reserved.
+//-
+
+div(ng-repeat="todo in todos")
+  input(type="checkbox" ng-click="makeDone(todo)")
+  span(ng-class="{'done': todo.done}") {{todo.description}}
+  a(href="#" ng-click="removeTodoItem(todo.id)") remove
```

Comment:
I think this would be better composed as:
``` jade
div(ng-repeat="todo in todos")
    todo-item(id="{{todo.id}}")
```

Then, you can move the contents of the loop into their own directive.